src/Vocup/MainForm.cs

```diff
@@ -125,9 +125,45 @@ public void UnloadBook(bool fullUnload)
             }
         }
 
+        private void StoreSettings()
+        {
+            //
+            // save the forms location and size
+            //
+            Settings.Default.MainFormLocation = this.Location;
+
+            if (this.WindowState == FormWindowState.Normal)
+            {
+                Settings.Default.MainFormSize = this.Size;
```

Comment:
I know it's a matter of preference, but I haven't used the `this` qualification in Vocup unless necessary. Please follow this style and remove the `this` keyword wherever possible.